doc/getting_started/setup_fpga.md

```diff
@@ -148,15 +148,15 @@ To ensure that you have sufficient access permissions, set up the udev rules as
 
 ## Flash the bitstream onto the FPGA
 
-To flash the bitstream onto the FPGA using the `cw310_loader` tool, use the following command:
+To flash the bitstream onto the FPGA using `opentitantool`, use the following command:
 
 ```console
 cd $REPO_TOP
 
 # If you downloaded the bitstream:
-./util/fpga/cw310_loader.py --bitstream build-bin/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
+bazel run //sw/host/opentitantool load-bitstream build-bin/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
```

Comment:
Or `./bazelisk sh`? Not sure what we want to use for canonical references.

Reply:
You're right, these should be `./bazelisk.sh ... ` to avoid surprises after a `git clone ...`

Reply:
I'd prefer bazel to keep it more readable. There's no way to get through getting_started without bazel getting installed or aliased to bazelisk.sh